performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -107,6 +107,18 @@ void ExperimentConfiguration::setup(int argc, char ** argv)
     "with_security", "Enables the security with ROS2")("roundtrip_mode",
     po::value<std::string>()->default_value("None"),
     "Selects the round trip mode (None, Main, Relay).")
+#ifdef ODB_FOR_SQL_ENABLED
```

Comment:
Prefix missing

Reply:
Do you mean I should change ODB_FOR_SQL_ENABLED to something like
`PERFORMANCE_TEST_ODB_FOR_SQL_ENABLED` or what do you mean with prefix?

Reply:
Yes, this prevents conflicts with 3rd party code and is also consistent with the rest of the code base.